.storybook/stories/useGLTF.stories.tsx

```diff
@@ -1,19 +1,20 @@
-import * as  React from 'react'
+import * as React from 'react'
+import { Vector3 } from 'three'
 
 import { Setup } from '../Setup'
 
-import { useGLTF } from '../../src/useGLTF'
+import { useGLTF } from '../../src'
 
 export default {
   title: 'Loaders/GLTF',
   component: useGLTF,
-  decorators: [(storyFn) => <Setup cameraPosition={[0, 0, 5]}>{storyFn()}</Setup>],
+  decorators: [(storyFn) => <Setup cameraPosition={new Vector3(0, 0, 5)}>{storyFn()}</Setup>],
 }
 
 function Suzanne() {
-  const { nodes, materials } = useGLTF('suzanne.glb', true)
+  const { scene } = useGLTF('suzanne.glb', true)
```

Comment:
`nodes` and `materials` are not valid keys of `THREE.GLTF` so this would fail.

I can't seem to find any core documentation on this, but if people agree we could impliment an extension to `THREE.GLTF` that would create the type with those keys. ***Very*** quick proposal:

```ts
interface GLTF extends THREE.GLTF {
  nodes: [key: string]: THREE.Object3D
  materials: [key: string]: THREE.Material
}
```